### PR TITLE
tidb-server: update server version

### PR DESCRIPTION
**Pull request by ngaut**

### CODE REVIEW

File: mysqldef/const.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ package mysqldef
 const (
 	MinProtocolVersion byte   = 10
 	MaxPayloadLen      int    = 1<<24 - 1
-	ServerVersion      string = "5.5.31-cm-1.0"
+	ServerVersion      string = "5.5.31-TiDB-1.0"
```
Comment (siddontang) [inline mysqldef/const.go]: 5.5.31 should be removed?

Comment (ngaut) [inline mysqldef/const.go]: We can't remove it, some driver may use it to check protocol version of MySQL.

Comment (siddontang) [inline mysqldef/const.go]: got it, maybe we can use 5.7.xx, because now all we refer is 5.7's manual.

Comment (ngaut) [inline mysqldef/const.go]: yep, Maybe later. 5.5.31 is safe for most cases we met.

**Comment by siddontang** [comment]
LGTM

**Comment by shenli** [comment]
LGTM

**Pull request merged**